### DISCUSSION TITLE
Individueller Druckmaßstab

### DISCUSSION THREAD

**Discussion by k-placer** [e-netze/webgis-community#63] [Q&A]
Unsere User*innen wollen in EPSG:3857 einen selbstdefinierten Maßstab in der Leiste links unten eintippen. Die Umrechnung erfolgt so, dass eine unrunder Zahl angezeigt wird. 

<add key="map-calc-crs" value="31259"/>

![image](https://user-images.githubusercontent.com/97947925/235649068-d12b5b41-a953-4bef-8bad-87e948eba13d.png)

In der custom.js haben wir daher folgenden Key hinzugefügt:
webgis.usability.zoom.useIterativeZoomingToFixScale = true;
![image](https://user-images.githubusercontent.com/97947925/235649188-914097eb-6e55-4ffc-bd4f-564e1e8218f3.png)

Jetzt wollen wir die Karte in 1:6.000 ausdrucken. Leider können wir aber beim Drucken keinen individuellen Maßstab mehr hinzufügen. Es wird immer auf den nächstgelegenen,  vorkonfigurierten Druckmaßstab gesprungen. 
![image](https://user-images.githubusercontent.com/97947925/235649607-b14a97ac-8479-4b45-86fc-c7cc5b63ba8c.png)

Gibt es eine Möglichkeit den Key zu setzen und trotzdem in individuellen Druckmaßstäben zu drucken?

**Comment by gis-eni** [comment]
Was muss ich machen um das Problem nachzuvollziehen?
Ich hab das bei mir Probiert => geht
In NÖGIS Atals probiert => geht
Browser: Edge, Chrome, FF

**Reply by k-placer** [reply]
Danke für die Rückmeldung! Das Problem hat sich geklärt. Es lag am Drucklayout und hatte nichts mit dem key zu tun.
